extensions/blocks/map/map.php

```diff
@@ -14,6 +14,47 @@
 	)
 );
 
+/**
+ * Fetch the A8C Mapbox access token from the WPCOM API.
+ *
+ * @return string
+ */
+function jetpack_fetch_mapbox_a8c_access_token() {
+	if ( ! Jetpack::is_active() ) {
+		return '';
+	}
+
+	$site_id = Jetpack_Options::get_option( 'id' );
+
+	$response = wp_remote_get( 'https://public-api.wordpress.com/wpcom/v2/sites/' . $site_id . '/mapbox' );
```

Comment:
Would it be useful to make requests using `wpcom_json_api_request_as_blog` instead? I am thinking that knowing where most of the requests come from may prove useful in some cases, when trying to analyze traffic?
https://github.com/Automattic/jetpack/blob/1b981c3573c82a4c83b3c212a241b59097691792/packages/connection/src/class-client.php#L349

Reply:
TIL!